packages/serverpod/lib/src/server/endpoint_dispatch.dart

```diff
@@ -228,7 +251,7 @@ abstract class EndpointDispatch {
     Map<String, dynamic> additionalParameters = const {},
   }) {
     if (descriptions.isEmpty) return {};
-
+    print('Parsing parameters: $paramString');
```

Comment:
Suggestion: remove debug print

Reply:
Woops 💅 I thought the analyzer caught those so that's why I didn't search for it 🧐